src/components/Add/AddSubmit.tsx

```diff
@@ -2,26 +2,57 @@ import React, { ReactElement, useEffect, useState } from "react";
 import { useLocation } from "react-router";
 import { useNavigate } from "react-router-dom";
 import "./Add.css";
+import { Storage } from 'aws-amplify';
 
 export default function AddSubmit(): ReactElement {
   const [status, setStatus] = useState<string>();
   const [check, setCheck] = useState<boolean>(false);
+  const [objectFileKey, setObjectFileKey] = useState<string>("");
+  const [imageMethod2Key, setImageMethod2Key] = useState<string>("");
+
+  const [objectFileUrl, setObjectFileUrl] = useState<string>("");
+  const [imageMethod2Url, setImageMethod2Url] = useState<string>("");
   const location: any = useLocation().state;
   const navigate = useNavigate();
 
   useEffect(() => {
     if (location.responseData.message.add_object_success) {
       setStatus("Success!");
+      setObjectFileKey(location.responseData.message.object_file_key);
+      setImageMethod2Key(location.responseData.message.image_method2_key);
     } else {
       setStatus("Fail!");
     }
-  }, []);
-
-  // location.responseData.message
+  }, [location.responseData.message.add_object_success, location.responseData.message.image_method2_key, location.responseData.message.object_file_key]);
 
   const handleHome = () => {
     navigate("/");
   };
+
+  const getFileUrl = async (key: string) => {
+    const fileUrl = await Storage.get(key);
+    return fileUrl;
+  };
+
+  useEffect(() => {
+    if(objectFileKey.length > 0) {
+      const getObjectFileUrl = async() => {
+        const urlFile = await getFileUrl(objectFileKey)
+        setObjectFileUrl(urlFile);
+      }
+      getObjectFileUrl()
+    }
+
+    if(imageMethod2Key.length > 0) {
+      const getImageMethod2Url = async() => {
+        const urlImageMethod2 = await getFileUrl(imageMethod2Key)
+        setImageMethod2Url(urlImageMethod2);
+      }
+      getImageMethod2Url()
+    }
+  }, [imageMethod2Key, objectFileKey]);
+
+  const typeImages = ["jpeg", "jpg", "png", "svg", "gif", "ico"];
```

Comment:
This is better to put outside of component

Reply:
Taras, thank you for checking my code) I'll fix everything)